### PR TITLE
Add --fix* flags, remove bomb

### PR DESCRIPTION
**Pull request by fejta**
/assign @cblecker @cjwagner

### CODE REVIEW

File: prow/cmd/peribolos/main.go [hunk 1]
```diff
@@ -82,6 +86,11 @@ func (o *options) parseArgs(flags *flag.FlagSet, args []string) error {
 	flags.IntVar(&o.tokensPerHour, "tokens", defaultTokens, "Throttle hourly token consumption (0 to disable)")
 	flags.IntVar(&o.tokenBurst, "token-burst", defaultBurst, "Allow consuming a subset of hourly tokens in a short burst")
 	flags.StringVar(&o.dump, "dump", "", "Output current config of this org if set")
```
Comment (cblecker) [inline prow/cmd/peribolos/main.go]: Should all these new flags be incompatible with dump? It seems odd that you'd want to take action, while also dumping a config of a specified org.

**Comment by stevekuznetsov** [comment]
/lgtm
/hold
Agreed that the flags should be disjoint with `--dump`

**Comment by fejta** [comment]
> be a dryrun or confirm setting

https://github.com/kubernetes/test-infra/blob/381ecf20efac306cca340ce607c7a294c0b1695e/prow/cmd/peribolos/main.go#L80

**Comment by fejta** [comment]
/hold cancel

Agree that `--fix-teams --dump` doesn't make a lot of sense, just like `--required-admins=fejta --maximumDelta=0.2 --require-self --confirm --dump` (all of which are irrelevant and will nonetheless not error). I don't really think adding all these unit tests is necessary though.

I added a `--dump ^ --config-path` check because one can reasonably expect this to:
a) Read config
b) Get current settings for the org
c) Append this org to the config

And I might want it to do this in the future, but it is more complicated so didn't bother for now.

Happy to accept a PR (or further argument that these checks are needed)

File: prow/cmd/peribolos/main.go (hunk 1 above)
Comment (fejta) [reply prow/cmd/peribolos/main.go]: Technically yes (and many of the older ones too)

**Pull request merged**